src/domain/core/src/services/pull_service.rs

```diff
@@ -172,17 +172,25 @@ pub trait PullMultiListener: Send + Sync {
 
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
 pub enum PullResult {
-    UpToDate,
+    UpToDate(UpToDateResult),
     Updated {
         old_head: Option<Multihash>,
         new_head: Multihash,
     },
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub enum UpToDateResult {
+    IngestUpToDate { uncacheable: bool },
+    UpToDate,
```

Comment:
I think it will be better to change this type to something like:
```rust
enum PullResultUpToDate {
  PollingIngest(PollingInsgestResultUpToDate),
  PushIngest(PushInsgestResultUpToDate),
  Transform(TransformResultUpToDate),
  Sync(SyncResultUpToDate),
}
```
This will require creating structs like `PollingInsgestResultUpToDate` in the lower-level services, but when we in future add more fields to them we will not have to touch the pull service - they be nicely wrapped already.

Also note the name `PullResultUpToDate` - when creating structs for enum variants I try to always use `EnumNameVariantName` convention, so this will be more consistent with the existing code.